src/main/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -136,68 +129,241 @@
         /** The fraction of bases that attained at least 90X sequence coverage in post-filtering bases. */
-        public double PCT_90X;
+         @NoMergingIsDerived
+         public double PCT_90X;
         /** The fraction of bases that attained at least 100X sequence coverage in post-filtering bases. */
-        public double PCT_100X;
+         @NoMergingIsDerived
+         public double PCT_100X;
 
         /** The theoretical HET SNP sensitivity. */
+        @NoMergingIsDerived
         public double HET_SNP_SENSITIVITY;
 
         /** The Phred Scaled Q Score of the theoretical HET SNP sensitivity. */
+        @NoMergingIsDerived
         public double HET_SNP_Q;
+
+        /**
+         * Merges the various PCT_EXC_* metrics.
+         * @param other metric to merge into this one.
+         */
+        @Override
+        public void merge(final MergeableMetricBase other) {
+            final WgsMetrics otherMetric = (WgsMetrics) other;
+
+            if (depthHistogram == null || otherMetric.depthHistogram == null) throw new PicardException("Depth histogram is required when deriving metrics.");
+
+            // NB:
+            // Since: PCT_EXC_TOTAL     = (totalWithExcludes - thisMetricTotal) / totalWithExcludes;
+            // Thus:  totalWithExcludes = total / (1 - PCT_EXC_TOTAL)
+            // Proof: Exercise is left to the reader.
+            final long thisMetricTotal        = (long) depthHistogram.getSum();
+            final long otherMetricTotal       = (long) otherMetric.depthHistogram.getSum();
+            final long total                  = thisMetricTotal + otherMetricTotal;
+            final long thisTotalWithExcludes  = (long) (thisMetricTotal / (1.0 - PCT_EXC_TOTAL));
+            final long otherTotalWithExcludes = (long) (otherMetricTotal / (1.0 - otherMetric.PCT_EXC_TOTAL));
+            final double totalWithExcludes    = thisTotalWithExcludes + otherTotalWithExcludes;
+
+            if (0 < totalWithExcludes) PCT_EXC_DUPE     = (PCT_EXC_DUPE * thisTotalWithExcludes + otherMetric.PCT_EXC_DUPE * otherTotalWithExcludes) / totalWithExcludes;
+            if (0 < totalWithExcludes) PCT_EXC_MAPQ     = (PCT_EXC_MAPQ * thisTotalWithExcludes + otherMetric.PCT_EXC_MAPQ * otherTotalWithExcludes) / totalWithExcludes;
+            if (0 < totalWithExcludes) PCT_EXC_UNPAIRED = (PCT_EXC_UNPAIRED * thisTotalWithExcludes + otherMetric.PCT_EXC_UNPAIRED * otherTotalWithExcludes) / totalWithExcludes;
+            if (0 < totalWithExcludes) PCT_EXC_BASEQ    = (PCT_EXC_BASEQ * thisTotalWithExcludes + otherMetric.PCT_EXC_BASEQ * otherTotalWithExcludes) / totalWithExcludes;
+            if (0 < totalWithExcludes) PCT_EXC_OVERLAP  = (PCT_EXC_OVERLAP * thisTotalWithExcludes + otherMetric.PCT_EXC_OVERLAP * otherTotalWithExcludes) / totalWithExcludes;
+            if (0 < totalWithExcludes) PCT_EXC_CAPPED   = (PCT_EXC_CAPPED * thisTotalWithExcludes + otherMetric.PCT_EXC_CAPPED * otherTotalWithExcludes) / totalWithExcludes;
```

Comment:
would it be onerous to include an IntervalList for each metric and make sure that they have zero intersection before merging? I worry that folks will try to collect this metric over random subsets of the data rather than distinct regions...